langchain/scripts/create-entrypoints.js

```diff
@@ -66,6 +66,7 @@ const entrypoints = {
   "llms/bedrock": "llms/bedrock",
   "llms/llama_cpp": "llms/llama_cpp",
   "llms/writer": "llms/writer",
+  "llms/portkey": "llms/portkey",
```

Comment:
After doing the above, you'll need to put `llms/portkey` under `requiresOptionalDependency`

Reply:
Placed this under the `requiresOptionalDependency`